packages/app/src/pages/rights/index.tsx

```diff
@@ -24,6 +24,7 @@ import {
   Table,
   TablePagination,
 } from '@frachtwerk/essencium-lib'
+import { hasRequiredRights } from '@frachtwerk/essencium-lib/src/utils/hasRequiredRights'
```

Comment:
This needs to be changed so we just need to import `@frachtwerk/essencium-lib` instead of the whole path. You need to export the function in the `index.ts` file inside the `utils` folder. The same is true for the other occurances.